db/migrate/20150513121721_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
Yep, your solution is way more secure. And yes I think you can remove it after you rollback your local database. I think your Postgres still keeps a track of migrations and this will most likely be the last - you can do `rake db:rollback` to make sure, or inspect the database itself and check respective schema_migrations table. Make sure you purge also test database and reset it.

If you know what you are doing, you can manipulate migrations. You are working alone on this project, so I can not see how bad it would be. Maybe you will have to force push to staging if you would have merged it and migrated there.